UPDATING.md

```diff
@@ -24,6 +24,11 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### HTTPHook verify default value changed from False to True.
+
+The HTTPHook is now secured by default: verify=True.
```

Comment:
I think it's worth adding a link to the class description. Now it is not possible in a clean way, but when my change is accepted, we should add a link and add a note under the parameter description.
Reference: 
https://github.com/apache/airflow/pull/4827/files

Reply:
Seems a bit excessive isn't it? Didn't see any other project that in it's change log adds links to each change class. Not to forget the people who reads this change log are programmers who are familiar with Airflow.

Reply:
I now have open documentation for boto3. This contains links to classes.  
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/upgrading.html
![Screenshot 2019-03-11 at 14 35 49](https://user-images.githubusercontent.com/12058428/54127851-07efdd00-440b-11e9-8741-c830c2472217.png)
It is not difficult to add linking. All you need to do is enter the full name of the class.
Example:
![Screenshot 2019-03-11 at 14 37 39](https://user-images.githubusercontent.com/12058428/54127952-3ff72000-440b-11e9-8d17-a0c6701495af.png)

Reply:
I do not want you to add a link now. I only think aloud plans for future changes and improvmenet.

Reply:
OK. So if anything else is needed let me know.
Currently there are no more comments so waiting for merge.